### PR TITLE
[REVIEW] Excluded vendor directory and framework code from scrutinizer

### PR DESCRIPTION
**Pull request by arothuis**
In order to get a good overview regarding *our* code, the vendor directory and in-project framework code should be excluded from Scrutinizer's scrutinisation.

### CODE REVIEW

**Comment by arothuis** [comment]
Review; @DRvanR, @rjkip

File: .scrutinizer.yml [hunk 1]
```diff
@@ -1,7 +1,12 @@
 filter:
     excluded_paths:
         - "*/Tests/*"
-
+        - "vendor/*"
+        - "app/SymfonyRequirements.php"
+        - "app/AppCache.php"
+        - "app/check.php"
+        - "web/app.php"
+        - "web/config.php"
```
Comment (DRvanR) [inline .scrutinizer.yml]: Shouldn't even be in the project root :wink:

File: .scrutinizer.yml [hunk 2]
```diff
@@ -1,7 +1,12 @@
 filter:
     excluded_paths:
         - "*/Tests/*"
-
+        - "vendor/*"
+        - "app/SymfonyRequirements.php"
+        - "app/AppCache.php"
+        - "app/check.php"
+        - "web/app.php"
```
Comment (DRvanR) [inline .scrutinizer.yml]: Why should this be ignored? It would be better to just solve the issues?

File: .scrutinizer.yml [hunk 3]
```diff
@@ -1,7 +1,12 @@
 filter:
     excluded_paths:
         - "*/Tests/*"
-
+        - "vendor/*"
+        - "app/SymfonyRequirements.php"
+        - "app/AppCache.php"
+        - "app/check.php"
```
Comment (DRvanR) [inline .scrutinizer.yml]: This file should be gitignored and thus not be part of the project?

File: .scrutinizer.yml [hunk 4]
```diff
@@ -1,7 +1,12 @@
 filter:
     excluded_paths:
         - "*/Tests/*"
-
+        - "vendor/*"
+        - "app/SymfonyRequirements.php"
```
Comment (DRvanR) [inline .scrutinizer.yml]: This file should be gitignored and thus not be part of the project?

File: .scrutinizer.yml [hunk 5]
```diff
@@ -1,7 +1,12 @@
...
+        - "app/AppCache.php"
```
Comment (DRvanR) [inline .scrutinizer.yml]: If we're not using it, why is it still in the project?

**Comment by arothuis** [comment]
Need to remove some Symfony bloat first. Vendor dirs are excluded by default in Scrutinizer. Closing PR.

**Pull request closed without merging**